### ISSUE TITLE
valid_docs: Possible false negative 

### ISSUE THREAD

**Issue by dtrenz** [realm/SwiftLint#561]
```swift
  /**
   Fetch the current value/state of a preference with the given identifier.

   - Parameter identifier: The identifier of the preference item

   - Returns: The current value of the preference
   */
  public static func get(identifier: String) -> AnyObject? {
```

...triggers this warning on the "Returns" line:

`Valid Docs Violation: Documented declarations should be valid. (valid_docs)`

**Comment by dtrenz** [comment]
Had an old version of swiftlint. Updating to 0.9.1 resolved this.
